### PR TITLE
[Critical Error Fix] FIX Cannot find module: 'lodash.isequalwith'.

### LINKED ISSUE #480: Cannot find module: 'lodash.isequalwith'.
When run npm install ,Failed to compile.
> ./node_modules/connected-react-router/esm/ConnectedRouter.js
Cannot find module: 'lodash.isequalwith'. Make sure this package is installed.

connected-react-router version is 6.9.0.

image

Module ***lodash.isequal*** should be  in dependencies rathe than devDependencies?

### PR DESCRIPTION
**Pull request by ltnscp9028**
The error occurred while modifying #408.
Fixed an error where lodash.isequalwith was duplicated in devDependency and peerDependency.
However, after removing it from devDependency, an error in the test process.
so,I moved lodash.isequalwidth to dependency. non peerDependency and devDependency.
fixes #480

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,11 +28,11 @@
     "posttest": "npm run lint"
   },
   "dependencies": {
+    "lodash": "^4.17.21",
```
Comment (Ivocin) [inline package.json:31]: use `"lodash.isequalwith": "^4.4.0"` is enough, so you don't need to change `ConnectedRouter.js`  file

Comment (ltnscp9028) [reply package.json:31]: Check this issue -> https://github.com/supasate/connected-react-router/pull/408 
It was caused by adding it to devDependecy, and if you remove it from devDependency, a test error occurs.
This is why I changed ```lodash.isequalwith``` to ```lodash```.

Comment (ltnscp9028) [reply package.json:31]: > use `"lodash.isequalwith": "^4.4.0"` is enough, so you don't need to change `ConnectedRouter.js` file

I think your intention is to install lodash.isequalwith only one installation in the Dependency, not peerDependency and devDependency. write ??

Comment (Ivocin) [reply package.json]: Yes, now the commit is OK.

**Review by Ivocin — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (ltnscp9028) [reply package.json]: @Ivocin Thanks for the suggestion that helped us make better code.🙏

**Review by robikovacs — APPROVED** [review]

**Comment by supasate** [comment]
I agree that we should bundle lodash.isequalwith instead of letting people install them separately to prevent this issue.

**Pull request merged**